src/NServiceBus.Core.Analyzer/AwaitOrCaptureTasksAnalyzer.cs

```diff
@@ -81,7 +81,12 @@ bool IsNServiceBusApi()
             "NServiceBus.Endpoint.Create",
             "NServiceBus.Endpoint.Start",
             "NServiceBus.IStartableEndpoint.Start",
-            "NServiceBus.IEndpointInstance.Stop");
+            "NServiceBus.IEndpointInstance.Stop",
+            "NServiceBus.UniformSession.IUniformSession.Send",
+            "NServiceBus.UniformSession.IUniformSession.Publish",
+            "NServiceBus.UniformSession.IUniformSessionExtensions.Send",
+            "NServiceBus.UniformSession.IUniformSessionExtensions.SendLocal",
+            "NServiceBus.UniformSession.IUniformSessionExtensions.Publish");
```

Comment:
We are missing our big shouty warning comment that we will not do this for any other downstream 😉.

Reply:
added a comment. Feel free to improve the wording.